logprep/util/configuration.py

```diff
@@ -816,9 +816,12 @@ def _verify(self):
         for processor_config in self.pipeline:
             try:
                 processor = Factory.create(deepcopy(processor_config))
+                processor.setup()
```

Comment:
That's a good start, but I don't think calling `setup` will suffice. The reason being that we often gather resources through `properties`. In the pseudonymizer for example the public keys are loaded not in the `setup` method but via the `_encrypter` property which again is called for the first time inside the `_pseudonymize` method. So with the current implementation I still see logging like this:
```
024-06-10 10:19:53 174478 Pipeline1  ERROR   : ProcessingCriticalError: ProcessingCriticalError: [Errno 2] No such file or directory: '/path/to/analyst_pubkey.pem' -> event was send to error output and further processing stopped, rule.id='ae29c373ba466990d81428cb2d8ff6253a35cb88dc37e9b4ef5e9669333f1806', rule.description='', event={'foo': 'bar'}
```

Reply:
fixed by calling all cached_properties during setup time